core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review.directive.html

```diff
@@ -85,7 +85,10 @@ <h4 class="col oppia-translation-modal-author-header">Submitted by <[authorName]
       Review message should be at most <[MAX_REVIEW_MESSAGE_LENGTH]> characters.
     </div>
   </div>
-  <div ng-show="!reviewable && suggestionIsRejected" class="oppia-suggestion-review-message">
+  <div ng-show="reviewMessage == '' && !reviewable && suggestionIsRejected" class="oppia-suggestion-review-message">
+    <i class="fas fa-spinner fa-spin" aria-hidden="true"></i>
+  </div>
+  <div ng-show="reviewMessage!='' && !reviewable && suggestionIsRejected" class="oppia-suggestion-review-message">
```

Comment:
```suggestion
  <div ng-show="reviewMessage !== '' && !reviewable && suggestionIsRejected" class="oppia-suggestion-review-message">
```

Prefer triple equals/not equals as double equals lead to error-prone type coercions. Here and above.

Reply:
Done